.changelog/unreleased/improvement/418-host-timestamp.md

```diff
@@ -0,0 +1,2 @@
+- Make ValidationContext::host_timestamp() abstract
```

Comment:
Since we removed the `pending_host_consensus_state` method, shouldn't we put this change under the "breaking changes" category? and also mention it here?

Reply:
absolutely [07f61e8](https://github.com/cosmos/ibc-rs/pull/457/commits/07f61e8f128559322dfcbce1e36c34f48e1f3382)